src/main/resources/DDL_CreateGpfdTablesMojfin.sql

```diff
@@ -50,5 +50,10 @@ DROP TABLE "GPFD"."REPORT_TRACKING";
 SET DEFINE OFF;
 Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REPORT_OWNER,REPORT_CREATOR,REPORT_DESCRIPTION,EXCEL_REPORT,EXCEL_SHEET_NUM,CSV_NAME,OWNER_EMAIL) values (1,'CCMS_invoice_analysis-CIS-to-CCMS-import-analysis-2','SELECT * FROM ANY_REPORT.V_CIS_TO_CCMS_INVOICE_SUMMARY',' https://justiceuk.sharepoint.com/:x:/r/sites/FinanceSysReference/Shared%20Documents/General/Monthly%20Accounts/Sharepoint%20base%20reports/General%20CCMS%20Tools/CCMS%20invoice%20analysis.xlsb?d=w7bc78b4b2c94489e899415353a37d234&csf=1&web=1&e=K0h3OD','Chancey Mctavish','Sophia Patel','Summary of invoices in CIS and CCMS by original source IT system','CCMS_invoice_analysis',2,'CIS-to-CCMS-import-analysis','owneremail@email.com');
 Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REPORT_OWNER,REPORT_CREATOR,REPORT_DESCRIPTION,EXCEL_REPORT,EXCEL_SHEET_NUM,CSV_NAME,OWNER_EMAIL) values (2,'CCMS_and_CIS_Bank_Account_Report_w_Category_Code_(MNTH)-MAIN-12','SELECT * FROM ANY_REPORT.V_BANK_MONTH','https://justiceuk.sharepoint.com/sites/FinanceSysReference/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FFinanceSysReference%2FShared%20Documents%2FGeneral%2FMonthly%20Accounts%2F2023%2D06%2FBank%20reporting&viewid=5d6ec327%2D2975%2D4d7c%2Dbd8d%2D6b793c45868b','Daniel Mctavish','Brian Limond','Summary of all payments made by CCMS/CIS and all cash receipts applied to debt in the previous month','CCMS_and_CIS_Bank_Account_Report_w_Category_Code_(MNTH)',12,'MAIN','secondowneremail@email.com');
+Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REPORT_OWNER,REPORT_CREATOR,REPORT_DESCRIPTION,EXCEL_REPORT,EXCEL_SHEET_NUM,CSV_NAME,OWNER_EMAIL) values (3,'CIS to CCMS import exceptions','SELECT * FROM ANY_REPORT.V_CIS_TO_CCMS_INVOICE_EXCEPTIONS','https://justiceuk.sharepoint.com/sites/FinanceSysReference/SitePages/GPFD-Test-Site-page.aspx','William Moran','William Unwin','Details of invoices transferred from CIS to CCMS by Legal Aid Scheme','CIS_TO_CCMS_IMPORT_EXCEPTIONS',3,'MAIN','William.Moran@Justice.gov.uk');
```

Comment:
Should we look to add all the new data rows into the data.sql file for testing purposes, when using H2 datastore?

Reply:
yes, all the database stuff should be tried out locally - think the order bug would have been caught with h2 testing too

Reply:
queries added